core/src/main/res/layout/fragment_logs.xml

```diff
@@ -14,10 +14,46 @@
    permissions and limitations under the License.
 -->
 
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:layout_margin="20dp"
     tools:context=".DevMenuLogsFragment">
 
-</FrameLayout>
\ No newline at end of file
+    <EditText
+        android:id="@+id/search_logs_field"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:hint="@string/search_logs_hint"
+        android:inputType="text"
+        android:autofillHints="no"
+        android:layout_toLeftOf="@+id/search_logs_button"
+        android:layout_toStartOf="@+id/search_logs_button"
+        android:layout_marginRight="20dp"
+        android:layout_marginEnd="20dp"
+        android:textSize="15sp"
+        android:textIsSelectable="true" />
+
+    <Button
+        android:id="@+id/search_logs_button"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/search_logs_button_text"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentRight="true" />
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="fill_parent"
+        android:layout_marginTop="15dp"
+        android:layout_below="@+id/search_logs_field">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/placeholder_logs"
+            android:textColor="@android:color/black"
```

Comment:
Instead of applying individual style customizations, consider applying a [system theme](https://developer.android.com/guide/topics/ui/look-and-feel/darktheme#supporting_dark_theme_in_your_app).